packages/api/src/driver/gitlab.ts

```
@@ -207,6 +210,200 @@ export class GitlabManager implements GitManager {
+          let file = null;
+          let fileName = '';
+          
+          // Try to find a README file
+          for (const filename of readmeFiles) {
+            try {
+              file = await this.gitlab.RepositoryFiles.show(identifier, filename, defaultBranch);
+              fileName = filename;
+              break;
+            } catch (error) {
+              // Continue to next filename if this one doesn't exist
+              continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)